src/styles/scss/_mixins.scss

```diff
@@ -0,0 +1,22 @@
+@mixin mobile {
+  @media screen and (min-width: 320px) {
```

Comment:
It would be great to set the values ​​of the min-width (320, 772, 1200) with variables

Reply:
done